components/bl-autocomplete-component/README.md

```diff
@@ -6,35 +6,38 @@ Autocomplete is a component of Backendless UI-Builder designer. This allows you
-
-### Events
-
-<dl>
-<dt>On autocomplete change</dt>
-<dd>Triggered when the user select an item from the options.</dd>
-<dt>On button clear click</dt>
-<dd>Triggered when the user clicks the clear button.</dd>
-<dt>On change</dt>
-<dd>Triggered when the user enters text into a text field.</dd>
-</dl>
+## Properties
+
+| Property             | Type                                                | Default value | Logic                      | Data Binding | UI Setting | Description                                                               |
+|----------------------|-----------------------------------------------------|---------------|----------------------------|--------------|------------|---------------------------------------------------------------------------|
+| Disabled             | *Checkbox*                                          | `false`       | Disabled Logic             | YES          | YES        | This handler allows you to disable a component.                                         |
+| Placeholder          | *Text*                                              | 'Placeholder' | Placeholder Logic          | YES          | YES        | This handler allows you to select a label of autocomplete component.                                         |
+| Autocomplete Variant | *Select* <br/> "outlined" \| "standart" \| "filled" | 'outlined'    | Autocomplete Variant Logic | YES          | YES        | This handler allows you to select the variant of autocomplete.                                      |
+| Options              | *JSON*                                              |               | Options Logic              | YES          | YES        | This handler allows you to add items to be displayed in the options list. Watch [Codeless Examples](#Examples). Signature of option `{objectId, value, label}` |
+
+## Events
+
+| Name                        | Triggers                                    | Context Blocks                             |
+|-----------------------------|---------------------------------------------|--------------------------------------------|
+| On Change Event             | when the user enters text into a text field | Input Value: `String`                      |
+| On Button Clear Click Event | when the user clicks the clear button       |                                            |
+
+## Styles
+
+**Theme**
+````
+@bl-customComponent-autocomplete-themeColor: @themePrimary;
+@bl-customComponent-autocomplete-backgroundColor: @appBackgroundColor;
+@bl-customComponent-autocomplete-textColor: @appTextColor;
+````
+
+**Dimensions**
+````
+@bl-customComponent-autocomplete-width: 300px;
+````
+
+## <a name="Examples"></a> Codeless Examples
+
+Adding options to options list:
+
+<img alt="adding options" src="./example-images/adding-options-to-options-list.png" width="720" />
```

Comment:
Change this example to sync codeless logic(create a list with objects), the user couldn't know what you have in **items** table.